exporters/jaeger/BUILD

```diff
@@ -180,6 +180,7 @@ cc_library(
     tags = ["jaeger"],
     deps = [
         ":jaeger_exporter",
+        "//sdk/src/common:global_log_handler",
```

Comment:
Do we need to add this for other exporters too. Eg, zipkin exporter includes sdk_config.h, which further includes global_log_handler.

Reply:
Exporters shoud denpend on `//sdk/src/logs` , `//sdk/src/metrics` or `//sdk/src/trace` , which already depend `//sdk/src/common:global_log_handler`